pkgs/tools/package-management/nix/common.nix

```diff
@@ -50,7 +50,11 @@ in
 , util-linuxMinimal
 , xz
 
-, enableDocumentation ? !atLeast24 || stdenv.hostPlatform == stdenv.buildPlatform
+, enableDocumentation ? !atLeast24 || (
+    (stdenv.hostPlatform == stdenv.buildPlatform) &&
+    # mdbook errors out on risc-v
+    !stdenv.buildPlatform.isRiscV
```

Comment:
This could get forgotten about and be left here long after the need for it has gone away. It might be better to change `mdbook` to exclude RiscV as a platform and then tryEval `mdbook` here. The restriction is then much more likely to be removed in a timely way after `mdbook` is fixed.

Reply:
mdbook does generally work on risc-v though. I tried building the test book included in the mdbook repo and it works. Not sure why it fails here

Reply:
Ping

Reply:
Please make this check based on if mdbook is supported/broken on the platform and change the platform/broken of mdbook accordingly.